src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests.cs

```diff
@@ -12174,5 +12174,50 @@ public static void Main(string[] args)
                 );
             var comp = CompileAndVerify(compilation, expectedOutput: "whatever");
         }
+
+        [Fact, WorkItem(12996, "https://github.com/dotnet/roslyn/issues/12996")]
+        public void TypeOfAVarPatternVariable()
```

Comment:
I'll open an issue for that request.

---
In reply to: [75774671](https://github.com/dotnet/roslyn/pull/13274#discussion_r75774671) [](ancestors = 75774671,75774455)